final_task/rss_reader/custom_error.py

```diff
@@ -0,0 +1,35 @@
+class Error(Exception):
+    """Base class for other exceptions"""
+
+
+class UrlNotFoundInArgsError(Error):
+    """Raised when can't find url in arguments"""
+    pass
+
+
+class NotEnoughArgumentsError(Error):
+    """Raised when not enough arguments"""
+    pass
+
+
+class NotValidUrlError(Error):
+    """Raised when url is not valid"""
+    pass
+
+
+class ConnectionFailedError(Error):
+    """Raised when can't connect to the url"""
+    message: str
+
+    def __init__(self, message):
```

Comment:
It is better to make `message` with default value like `message=None`
Same for other similar exception classes